package.json

```diff
@@ -92,8 +96,10 @@
     "fs-extra": "5.0.0",
     "jszip": "3.1.5",
     "minimist": "1.2.3",
+    "mkdirp": "^1.0.4",
     "mocha": "5.0.0",
     "prettier": "^2.0.5",
+    "rimraf": "^3.0.2",
```

Comment:
These dependencies allow commands `mkdir -p` and `rm -rf` to be cross-platform (Windows, Mac, Linux).